recipe-server/normandy/base/api/views.py

```diff
@@ -54,6 +54,19 @@ def get(self, request):
         return Response(UserSerializer(request.user).data)
 
 
+class ServiceInfoView(APIView):
```

Comment:
A docstring mentioning that this is used for sending data to the frontend would be useful.